test/integration/controller/core/workload_controller_test.go

```diff
@@ -171,6 +173,7 @@ var _ = ginkgo.Describe("Workload controller", func() {
 		ginkgo.AfterEach(func() {
 			gomega.Expect(framework.DeleteNamespace(ctx, k8sClient, ns)).To(gomega.Succeed())
 			gomega.Expect(framework.DeleteRuntimeClass(ctx, k8sClient, runtimeClass)).To(gomega.Succeed())
+			gomega.Expect(framework.DeleteResourceFlavor(ctx, k8sClient, flavor)).To(gomega.Succeed())
```

Comment:
why not reuse the same flavor in all tests cases? without ever deleting it?

Reply:
@alculquicondor If this flavor is not removed here, we face the error in `test/integration/controller/core/clusterqueue_controller_test.go` like the following since the 'spot' flavor is also used in integration tests for the clusterqueue-controller.

```sh
ClusterQueue controller when Reconciling clusterQueue usage status [It] Should update status when workloads are assigned and finish
/Users/tenzen-y/go/src/github.com/kubernetes-sigs/kueue/test/integration/controller/core/clusterqueue_controller_test.go:109

  [FAILED] Expected success, but got an error:
      <*errors.StatusError | 0x140004ad4a0>: {
          ErrStatus: {
              TypeMeta: {Kind: "", APIVersion: ""},
              ListMeta: {
                  SelfLink: "",
                  ResourceVersion: "",
                  Continue: "",
                  RemainingItemCount: nil,
              },
              Status: "Failure",
              Message: "resourceflavors.kueue.x-k8s.io \"spot\" already exists",
              Reason: "AlreadyExists",
              Details: {
                  Name: "spot",
                  Group: "kueue.x-k8s.io",
                  Kind: "resourceflavors",
                  UID: "",
                  Causes: nil,
                  RetryAfterSeconds: 0,
              },
              Code: 409,
          },
      }
      resourceflavors.kueue.x-k8s.io "spot" already exists
  In [It] at: /Users/tenzen-y/go/src/github.com/kubernetes-sigs/kueue/test/integration/controller/core/clusterqueue_controller_test.go:176 @ 12/19/22 02:39:00.155

  There were additional failures detected.  To view them in detail run ginkgo -vv
```

Reply:
I will remove this process to delete flavor for the same reason as https://github.com/kubernetes-sigs/kueue/pull/415/files#r1051645138.

```
I added flavor to the top of the ginkgo spec since I tested the ClusterQueueStatus.Conditions in workload_controller_test.go.

Although we can remove these flavors since we can ignore the ClusterQueueStatus.Conditions as you mentioned in the following:

https://github.com/kubernetes-sigs/kueue/pull/415#discussion_r1046096792
```

Reply:
Done.

Reply:
Ah, right. We can't create an object for a set of tests... just for the entire suite with `BeforeSuite`